tools/build_file_generator/third_party_deps/third_party_deps_updater.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# A script for updating third party dependencies inside third_party_deps_tool java_binary
+
+readonly REPO_ABS_PATH=$(git rev-parse --show-toplevel)
+# TODO: Make the getting of the path independent of the project
+readonly BUILD_FILE_ABS_PATH="$REPO_ABS_PATH/tools/build_file_generator/third_party_deps/BUILD"
+
+# Getting public third-party target names
+third_party_folder_path="$REPO_ABS_PATH/third_party"
```

Comment:
Why is this not `readonly` like the others, and styles as a constant?

Reply:
This variable is never used, I forgot to remove it. 
Removed this line.